lib/Attribute.js

```diff
@@ -419,11 +419,11 @@ Attribute.prototype.toDynamo = async function (val, noSet, model, options) {
     isExpires = true;
   }
 
-  if (isTimestamp && options.updateTimestamps === false) {
+  if (isTimestamp && (!options || options.updateTimestamps === false)) {
```

Comment:
I’m on my phone so can’t look at this super in depth. Is this changing existing behavior tho? Would this be considered a breaking change? Like if the user doesn’t pass in options, it looks like this might be changing the behavior of that.

Reply:
This function doesn't normally get called by user directly, and currently if it is called without passing options, this expression throws error when trying to access `updateTimestamps` field on `undefined`. I also made sure to update these expressions in a way that not passing options results in the same behavior as passing options without `updateTimestamps` and `updateExpires` specified, so that if a call to this function broke in some other place, it would now be fixed with correct behavior.

Reply:
I've just found one such "other place" and mentioned it in #384.